src/test/java/teammates/test/cases/action/InstructorCourseStudentDetailsEditSaveActionTest.java

```diff
@@ -58,7 +58,8 @@ public void testExecuteAndPostProcess() throws Exception {
                 Const.ParamsNames.STUDENT_NAME, student1InCourse1.name,
                 Const.ParamsNames.NEW_STUDENT_EMAIL, newStudentEmail,
                 Const.ParamsNames.COMMENTS, newStudentComments,
-                Const.ParamsNames.TEAM_NAME, newStudentTeam
+                Const.ParamsNames.TEAM_NAME, newStudentTeam,
+                Const.ParamsNames.SESSION_SUMMARY_EMAIL_SEND_CHECK, "true"
```

Comment:
Check also that even with this parameter set to `true`, requests that do not change the student email will not end up sending the email.